pkgs/tools/misc/owofetch/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "owofetch";
+  version = "0.3.1";
+
+  src = fetchFromGitHub {
+    owner = "netthier";
+    repo = "owofetch-rs";
+    rev = "v${version}";
+    sha256 = "sha256-I8mzOUvm72KLLBumpgn9gNyx9FKvUrB4ze1iM1+OA18=";
+  };
+
+  cargoSha256 = "sha256-Rd3pMWAnNdUalDc/f1019Tiyn73JukV3K7rrXOf/INk=";
```

Comment:
Hm, I tried to build this and got a hash mismatch, does this build for you?

```
error: hash mismatch in fixed-output derivation '/nix/store/wny079ma72sn6cbhibsdvh24b7mmhrwc-owofetch-0.3.1-vendor.tar.gz.drv':
         specified: sha256-Rd3pMWAnNdUalDc/f1019Tiyn73JukV3K7rrXOf/INk=
            got:    sha256-rfN4QERs1H1G7ZZim//78vlxbYfU4Cx7SYYUz/QLKeU=
```

Reply:
It was building when I tested earlier, but I'm now getting the same mismatch.

Reply:
Pushed a hash change.